kolibri/plugins/user/assets/src/state/actions.js

```diff
@@ -121,11 +122,21 @@ export function showSignInPage(store) {
     });
     Lockr.set(SIGNED_OUT_DUE_TO_INACTIVITY, null);
   }
-  resetAndSetPageName(store, {
-    pageName: PageNames.SIGN_IN,
-    title: translator.$tr('userSignInPageTitle'),
-  });
-  store.dispatch('SET_PAGE_STATE', {});
+
+  FacilityResource.getCollection()
```

Comment:
Given that this is happening on every load of the user page, I think we should make a cached bootstrap of the Facility collection into the `user.html` template.

Will open a PR shortly.